src/js/models/list-section.js

```diff
@@ -0,0 +1,68 @@
+import Section from './markup-section';
+import LinkedList from '../utils/linked-list';
+
+export const LIST_SECTION_TYPE = 'list-section';
+
+export default class ListSection extends Section {
+  constructor(tagName, items=[]) {
+    super(tagName);
+    this.type = LIST_SECTION_TYPE;
+
+    // remove the inherited `markers` because they do nothing on a ListSection but confuse
+    this.markers = undefined;
```

Comment:
Yeah, agreed. I was hoping to see more of an opportunity to build these sections up through composition rather than more granular inheritance levels, but using a `Markerable` section type does seem worthwhile.